src/eradiate/radprops/_afgl1986.py

```diff
@@ -22,6 +22,7 @@
 from ..thermoprops.util import (
     compute_column_mass_density,
     compute_column_number_density,
+    compute_volume_mixing_ratio_at_surface,
```

Comment:
Side comment: I think this naming pattern ("compute...") is heavy and could be made more concise. After all, the fact that a computation occurs is made obvious, and this actually breaks the expressiveness of code using your functions. For example, compare
```python
x = compute_volume_mixing_ratio_at_surface(...)
```
and
```python
x = volume_mixing_ratio_at_surface(...)
```
This is much nicer to read IMO.